utils/emoji_utils.tsx

```diff
@@ -112,3 +114,72 @@ export function wrapEmojis(text: string): React.ReactNode {
     // Only return an array if we're returning multiple nodes
     return nodes.length === 1 ? nodes[0] : nodes;
 }
+
+// Note : This function is not an idea implementation, a more better and efficeint way to do this come when we make changes to emoji json.
+export function convertEmojiSkinTone(emoji: SystemEmoji, newSkinTone: string): SystemEmoji {
+    let newEmojiId = '';
+
+    if (!emoji.skins && !emoji.skin_variations) {
```

Comment:
This now incorporates some of the logic from the previous version of this PR that avoids converting emojis that can't be converted